app/controllers/two_factor_authentication/otp_expired_controller.rb

```diff
@@ -22,5 +24,17 @@ def redirect_unless_otp_expired_redirect_enabled
     def otp_delivery_preference
       current_user.otp_delivery_preference
     end
+
+    def authentication_options_path
+      if mfa_context.enabled_mfa_methods_count < 1
```

Comment:
If the user were setting up a phone as their second MFA during account creation, would this logic cause them to be shown a link directing them to `login_two_factor_options_url` ? I think we only want to link them there when the user is trying to sign in.

Reply:
I had this concern as well, so asked about this in a [slack thread] (https://gsa-tts.slack.com/archives/C01710KMYUB/p1677707031189029). I will push back on this today and record the final decision.